pkg/rpc/server/server_test.go

```diff
@@ -701,6 +718,18 @@ func TestRPC(t *testing.T) {
 		assert.Equal(t, "AZ81H31DMWzbSnFDLFkzh9vHwaDLayV7fU", txOut.Address)
 	})
 
+	t.Run("gettransactionheight", func(t *testing.T) {
```

Comment:
I think you're better using some non-zero-block transaction and putting this test into `rpcTestCases`, we know our chain, so it shouldn't be a problem.